src/Layouts/Base.inc

```diff
@@ -133,7 +133,7 @@ Set_Comparator_Phase_A MACRO
 IF MCU_TYPE == MCU_BB1 or MCU_TYPE == MCU_BB2
     mov  CMP_MX, #((A_Mux SHL 4) + V_Mux)
 ELSEIF MCU_TYPE == MCU_BB51
-    mov  CMP_MX, #12h
+    mov  CMP_MX, #(10h+A_Mux-1)
```

Comment:
Spaces missing before and after `+` & `-` are missing.

Reply:
The format has been adjusted.